bahmni-docker/README.md

```diff
@@ -42,6 +42,9 @@ Once you have Docker installed, ensure that you are running the daemon. If you w
 
 Currently Bahmni has been tested on **docker-compose version 1.29.2**. If you are using older versions of docker-compose, please upgrade to the latest version. You can check docker compose version by running `docker-compose version`
 
+## Client Side Logging
+Client side logging has been configured using python-flask. To build a fresh image, set the `CLIENT_SIDE_LOGGING_PATH` to the path containing [client_side_logging](https://github.com/Bahmni/client_side_logging) files and volume mount the same in [docker-compose.yml](https://github.com/Bahmni/bahmni-package/blob/master/bahmni-docker/docker-compose.yml#L7)
```

Comment:
I feel that we need to build the proxy image along with client side logging code. The implementer doesn't want to volume mount the codebase since there is no implementation specific config or change that needs to be done. Also logging is something that we need to ship out of the box and it is not optional